### ISSUE TITLE
Tutanota choco package stopped auto-updating

### ISSUE THREAD

**Issue by wireddown** [A-d-r-i/update_choco_packages#15]
I'm using the choco package you maintain for **Tutanota**, and it appears the auto-updater stopped following the upstream releases.

- choco has version `3.109.13` as the latest -- https://community.chocolatey.org/packages/tutanota
- Tutanota has version `3.110.1` as the latest -- https://github.com/tutao/tutanota/releases

I'm creating this issue because the choco notes for Tutanota say

> Please Note: This is an automatically updated package. If you find it is out of date by more than a day or two, please contact the maintainer(s) and let them know the package is no longer updating correctly.

**Comment by A-d-r-i** [comment]
Hey!

Thank you very much for bringing the problem to my attention! Indeed the package was absolutely not up to date. The method I used to get new updates was flawed.

It's fixed with [855649c](https://github.com/A-d-r-i/update_choco_packages/commit/855649cc53b4b8fe45fd0614251a6d3bda4b9e87), you can download the new version right now : [[Tutanota on Choco](https://community.chocolatey.org/packages/tutanota/3.110.1)]

> I'm creating this issue because the choco notes for Tutanota say

This is the right solution, don't hesitate!

Thanks again! 😊

**Comment by wireddown** [comment]
It's live! Thank you 😄

```txt
$ choco outdated | grep tuta
tutanota|3.109.13|3.110.1|false
```
